src/test/ui/pattern/move-ref-patterns/borrowck-move-ref-pattern.stderr

```diff
@@ -187,21 +187,17 @@ LL |     drop(_x3);
    |          --- mutable borrow later used here
 
 error[E0382]: use of moved value: `tup`
-  --> $DIR/borrowck-move-ref-pattern.rs:43:14
-   |
-LL |       let mut tup = (U, U, U);
-   |           ------- move occurs because `tup` has type `(U, U, U)`, which does not implement the `Copy` trait
-LL |       let c1 = || {
-   |                -- value moved into closure here
-LL |           let (ref _x0, _x1, _) = tup;
-   |                                   --- variable moved due to use in closure
-LL |       };
-LL |       let c2 = || {
-   |  ______________^
-LL | |
-LL | |         let (ref mut _x0, _, _x2) = tup;
-LL | |     };
-   | |_____^ value used here after move
+  --> $DIR/borrowck-move-ref-pattern.rs:45:37
+   |
+LL |     let mut tup = (U, U, U);
+   |         ------- move occurs because `tup` has type `(U, U, U)`, which does not implement the `Copy` trait
+LL |     let c1 = || {
+   |              -- value moved into closure here
+LL |         let (ref _x0, _x1, _) = tup;
+   |                                 --- variable moved due to use in closure
+...
+LL |         let (ref mut _x0, _, _x2) = tup;
+   |                                     ^^^ value used here after move
```

Comment:
This part still differs from the expected diagnostic output

Reply:
My thoughts on why this doesn't work is that the fake reads are being added outside building the clsoure, so the borrow checker can't really figure out that the use is because of a closure capture. 

Not sure if this is something we want but maybe we add FakeReadLetClosure(closure_def_id) or something similar

Reply:
Another idea (don't think it's neccesarily a good one) that we have the usage span that is passed to the cfg be a multispan so that it points to both the closure and use within the closure